packages/thirdweb/src/chainsaw/formatter.ts

```diff
@@ -0,0 +1,132 @@
+import { formatBlock, formatTransaction } from "viem";
+import { numberToHex } from "../utils/encoding/hex.js";
+import { parseNFT } from "../utils/nft/parseNft.js";
+import type {
+  Block,
+  ChainsawBlock,
+  ChainsawEvents,
+  ChainsawNFTs,
+  ChainsawTransaction,
+  ChainsawTransactions,
+  Events,
+  NFTs,
+  Transactions,
+} from "./types.js";
+
```

Comment:
Could you add @internal to all these just so they're accounted for

Reply:
I assume you meant the functions, not the types. Done in [cca8c79](https://github.com/thirdweb-dev/js/pull/4004/commits/cca8c79d6a797aeed7cfe96585488632e1f532d9)